src/app.js

```diff
@@ -455,10 +455,13 @@ var run = function () {
 
   function editorOnChange () {
     var input = editor.getValue()
+    // save contents
+    editor.setCacheFileContent(input)
```

Comment:
Sorry, that was a typo, I did not mean 5 minutes.
I think we should save before each compilation and roughly once per second, depending on performance.